### ISSUE TITLE
Suggestion delete Polymicrogyria, anterior to posterior gradient

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4569]
Related to #4382
https://hpo.jax.org/app/browse/term/HP:0006821

THis term is currently used to annotate only one disease
https://www.omim.org/entry/606854

Renske, how should we annotate this disease?

**Comment by Renske-Oegema** [comment]
frontal-predominant cobblestone malformation

**Comment by Renske-Oegema** [comment]
PMID 9763366;     17878207;     19067347

**Comment by pnrobinson** [comment]
see also #4561 -- can we relabel this term? @Renske-Oegema

**Comment by pnrobinson** [comment]
duplicate with #4561, closing
